### PR TITLE
Update versions for periodic nightly & stable tests

### PR DESCRIPTION
**Pull request by creydr**
Need to update the versions for the periodic nightly & stable tests

### CODE REVIEW

File: ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml [hunk 1]
```diff
@@ -69,11 +69,11 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.10"
+      version: "4.11"
   stable:
     release:
       channel: stable
-      version: "4.9"
+      version: "4.11"
```
Comment (cybertron) [inline ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml:76]: Should this be 4.10?

**Comment by cybertron** [comment]
The failures in the handler-sdn-ipv4 jobs are related to the bug the nmstate team is looking into, right? The other job failures look like flakes, but every test failed in that one.

One question inline too.

**Comment by creydr** [comment]
> The failures in the handler-sdn-ipv4 jobs are related to the bug the nmstate team is looking into, right? The other job failures look like flakes, but every test failed in that one.

Yes. This is expected but do the nmstate bug. With this PR the current nightlies will start to fail too (due to this bug).

File: ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml (hunk 1 above)
Comment (creydr) [reply ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml:76]: We could set it to 4.10 as no stable 4.11 is available yet. But so we would need to update it in a few weeks again. But I am not very opinionated on this...

Comment (cybertron) [reply ci-operator/config/openshift/kubernetes-nmstate/openshift-kubernetes-nmstate-release-4.11.yaml:76]: Ah, okay. I was thinking this should match master, but I suppose master will get updated when 4.12 opens so it doesn't necessarily need to.

**Review by cybertron — COMMENTED** [review]
This lgtm, but do we know how it got out of sync? I thought the job configs were updated automatically as each release starts.

**Comment by creydr** [comment]
> This lgtm, but do we know how it got out of sync? I thought the job configs were updated automatically as each release starts.

I don't know. Maybe (but that's a big guess) it was because the 4.11 file was copied from the 4.10 which still referenced 4.9 as stable (https://github.com/openshift/release/pull/28195/files/fa041655e752585e4005904b14bcaccbe625357c#diff-cd116de8a5bd2e87471a6c9d60933baccfd7d9e8af93a12d17ac01e59e692565L79) and so the sync didn't know what we want to be updated to :man_shrugging:

**Comment by cybertron** [comment]
/lgtm

Okay, I guess we'll have to keep an eye on it in the future.

**Pull request merged**